atc/metric/emitter/newrelic.go

```diff
@@ -1,41 +1,74 @@
 package emitter
 
 import (
+	"bufio"
 	"bytes"
+	"compress/gzip"
 	"encoding/json"
 	"fmt"
+	"github.com/concourse/flag"
+	"io"
+	"io/ioutil"
 	"net/http"
 	"strings"
+	"sync"
 	"time"
 
 	"code.cloudfoundry.org/lager"
 	"github.com/concourse/concourse/atc/metric"
 	"github.com/pkg/errors"
 )
 
+const (
+	NEWRELIC_PAYLOAD_MAX_SIZE            = 1024 * 1024
```

Comment:
nit: trying to follow common conventions (e.g., Go std lib), we prefer camelcase 😁  e.g.: https://github.com/concourse/concourse/blob/7a6048b7d6a9124c891b58d9560a9af8be80edaf/tsa/commands.go#L3-L9

also, do these need to be exported? If I understood correctly, they're not necessary anywhere else 🤔

Reply:
Good catch! Definitely we should follow the convention.